scripts/generateESLintTest.js

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
This way, as your developing on the library, you have one place to look for output. I've found its overly optimistic to assume people will run eslint locally, and not just let CI do it. This way, you can run your tests in a browser, and see if everything is functionally good, and the code is clean. You end up with fewer people waiting on CI for linter output this way.